next.config.js

```diff
@@ -14,7 +14,14 @@ const nextConfig = {
   inlineImageLimit: 48, // make it tiny so that it doesn't inline,
   devIndicators: {
     autoPrerender: false, // If you need to know the page is correctly pre rendering enable this
-  }
+  },
+  typescript: {
+    /**
+     * This is temporary, until all JS code is ported to TS we 
+     * need to ignore typescript errors on building
+     */
+    ignoreBuildErrors: true,
```

Comment:
It's important that we don't forget this!

Reply:
Yes, the errors are there at development to be corrected, this affects only for the production build